strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/io/ArtifactOutputStreamTest.java

```diff
@@ -42,13 +42,22 @@
 {
 
     public static final String REPOSITORY_RELEASES = "aos-releases";
+    
+    private static boolean initialized;
```

Comment:
@fuss86 I have added this by analogue that you made [here](https://github.com/strongbox/strongbox/pull/237/files#diff-e3d73264f8d9b47f5b1ae6a2030ef695). As I understand `initialized` was used to step around something like `AccessDeniedExceptions`, which I have got now. I think we should find the cause of the Windows problems and resolve it, to prevent such problems in the future.